src/router/index.js

```diff
@@ -110,6 +111,14 @@ export default new Router({
         realm: 'issueviewer',
       },
     },
+    {
+      path: '/titles/:title_uid?',
```

Comment:
hehe, why not just "newspapers" :) title is misleading 
https://en.wikipedia.org/wiki/Newspaper

Reply:
fixed in commit b0e3c79